quiz/quiz.go

```diff
@@ -139,10 +134,11 @@ func (q *Quiz) Start(maxQuizTime int) error {
 // timeQuiz waits on the maxQuizTime and sends a "timeout" string down the channel when it is expired.
 // takeAndTimeQuiz returns the first thing down the channel, so the maxQuizTime will 'interrupt' the quiz if
 // it's still going
-func timeQuiz(quizTimer *time.Timer, c chan string) {
-	<-quizTimer.C
-	fmt.Println("\nTimer has run out!")
-	c <- "timeout"
+func timeQuiz(maxQuizTime int, c chan string) {
```

Comment:
You should probably pass in a duration here if you still want to pass something in here. (durations implement the Stringer interface so print statements will have units)